x-pack/platform/plugins/shared/streams_app/public/components/streams_list/index.tsx

```diff
@@ -216,6 +216,7 @@ function StreamNode({
         alignItems="center"
         className={css`
           padding: ${euiThemeVars.euiSizeXS};
+          padding-left: ${euiThemeVars.euiSizeS};
```

Comment:
`euiThemeVars` is discouraged by the EUI team, we should get the theme using the `useEuiTheme` hook for these variables.